README.md

```diff
@@ -93,7 +100,3 @@ There are some useful tools to post-process outputs of the pipeline.
 ### qc_jsons_to_tsv
 
 [This tool](utils/qc_jsons_to_tsv/README.md) recursively finds and parses all `qc.json` (pipeline's [final output](docs/example_output/v1.1.5/qc.json)) found from a specified root directory. It generates a TSV file that has all quality metrics tabulated in rows for each experiment and replicate. This tool also estimates overall quality of a sample by [a criteria definition JSON file](utils/qc_jsons_to_tsv/criteria.default.json) which can be a good guideline for QC'ing experiments.
```

Comment:
I like to be a bit wordy so- this is my opinion:
I think this is another very good feature to highlight. So may be making it more descriptive of how this is again a super-cool feature and how one could use this tool to have a first glance of "which are the good experiments vs which ones are bad"?
Also, some explanation on what would a "good pass experiment" and which would be a "not so good" or "bad" experiment entail. That is, how could the user use this output to get a sense of whether it was an experiment that they should continue working on or they should just discard this because it is not good- etc. I also do agree that the "user" who is using this pipeline will already have some numbers in mind but it is not a bad idea to give some way to gauge their experiment right here and hopefully they can save some time by not having to go to find these numbers at some other place.